style/index.css

```diff
@@ -36,7 +36,7 @@
   flex-direction: column;
   background: var(--jp-layout-color1);
   color: var(--jp-ui-font-color1);
-  font-size: var(--jp-ui-font-size0);
+  /* font-size: var(--jp-ui-font-size0); */
```

Comment:
This font size is for consistency with the title attributes of the other tabs (such as the running or command palette tab). Can we restore it?